meteor/client/lib/ReactMeteorData/ReactMeteorData.tsx

```diff
@@ -364,12 +365,12 @@ export function useSubscription<K extends keyof AllPubSubTypes>(
  */
 export function useSubscriptions<K extends keyof AllPubSubTypes>(
 	sub: K,
-	argsArray: Parameters<AllPubSubTypes[K]>[]
+	argsArray: Array<Parameters<AllPubSubTypes[K]> | undefined | null>
```

Comment:
add `false` as an option here too, so I can do

```
const component = (ready: boolean, studioId: StudioId) => {
	useSubscriptions(MeteorPubSub.buckets, [ready && [studioId, null]]) // won't work otherwise, Type 'false | any[]' is not assignable to type 'unknown[] | null | undefined'
}
```